lib/mongodb.js

```diff
@@ -431,7 +431,7 @@ MongoDB.prototype.toDatabase = function(model, data) {
   convertDecimalProps(data, props);
   // Override custom column names
   data = this.fromPropertyToDatabaseNames(model, data);
-  debug('toDatabase data: ', util.inspect(data, {depth: null}));
+  debug('toDatabase data: ', util.inspect(data));
```

Comment:
This is the case that we should use: 
```ts
if (debug.enabled) debug('toDatabase data: ', util.inspect(data));
```

Otherwise, `util.inspect` is always executed, which is pretty expensive even when debug is not turned on.